### PR TITLE
Add WWSK content section to website

### LINKED ISSUE #34: WorldWindServerKit website section
Candidate sections:

- Overview
    - Purpose
        - Vision/Scope
    - Features
        - GeoPackage
        - Integrated GeoWebCache
        - GDAL Formats

- Developer
    - Architecture/Design
        - Module descriptions
        - UML diagrams
    - How to build
        - Maven
        - Github

- User
    - Documentation
        - Export GeoPackage
        - WorldWind Preview
        - Out-of-the-box settings
        - Common Issues/FAQ, e.g., CORS 
    - How to add features
    - How to install/configure
        - Standalone/Jetty
        - Tomcat
        - Memory allocation
        - JAI 
    - How to interface with clients

### PR DESCRIPTION
**Pull request by eirizarry**
Closes #34

This PR will add a section for the WorldWind Server Kit to our website. 

Content includes: 
- Home
- Get Started
- Features
- Examples (under construction)
- Docs

This iteration is designed to meet the requirements for presenting WWSK at EO Open Science. The "Docs" page only contains information on WWSK modules for now. There is no content on "Examples", except for a disclaimer that the page is under construction. If there is no need to add examples right now, I will remove it. I believe there is more documentation to be added; my suggestion is to merge this now and submit new documentation through a new PR.

### CODE REVIEW

File: content/serverkit/_index.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+title: "WorldWind Server Kit"
+date: 2017-09-13T12:20:20-07:00
+draft: false
+---
+
+### What is WWSK?
+
+WWSK *(pronounced whis-kee)* stands for WorldWind Server Kit. It is an open source Java project that assembles
+[GeoServer](http://geoserver.org/) for easy data distribution and implementation. It is designed with simplicity in mind.
```
Comment (emxsys) [inline content/serverkit/_index.md]: Please strike the word 'data' in "for easy data distribution"

File: content/serverkit/docs/modules.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+[WorldWind GeoServer Application](https://github.com/NASAWorldWind/WorldWindServerKit/tree/develop/worldwind-geoserver)
+
+[WorldWind GeoServer Distribution](https://github.com/NASAWorldWind/WorldWindServerKit/tree/develop/worldwind-geoserver-dist)
+
+[WorldWind GeoPackage Extension](https://github.com/NASAWorldWind/WorldWindServerKit/tree/develop/worldwind-gs-geopkg)
+
+[WorldWind GeoPackage Module](https://github.com/NASAWorldWind/WorldWindServerKit/tree/develop/worldwind-gt-geopkg)
+
+[WorldWind GeoServer WMS Module](https://github.com/NASAWorldWind/WorldWindServerKit/tree/develop/worldwind-gs-wms)
```
Comment (emxsys) [inline content/serverkit/docs/modules.md]: There is an additional module to add: WorldWind GeoServer Viewer (or Explorer, TBD). It's under development in the feature/#107-worldwind_viewer branch

File: content/serverkit/features.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+- Understands multiple data formats such as Shapefiles, GeoTIFFS, GeoPackages, and GDAL formats
+- Serves and shares data with clients
+- Accesses data from remote services, acting as a proxy
+- Uses an integrated GeoWebCache for improved client performance
+
+## WorldWind Viewer
+
+- Allows you to see your data in the contet of a globe
+- Visualizes all the layers/layer groups that you've defined on one terrain
```
Comment (emxsys) [inline content/serverkit/features.md]: Please change "layers/layer groups" to "layers and layer groups"

**Review by emxsys — COMMENTED** [review]
Just a few minor edits are needed.

**Comment by emxsys** [comment]
Please keep the Examples section as I think it will be applicable, for instance, how to setup a Cascaded WMS to WorldWind servers.

**Review by pdavidc — CHANGES_REQUESTED** [review]
This is coming along very nicely. I see a need for a few changes:

- Get Started, Features, Examples, and Docs don't have the project header, which I found disorienting.
- Get Started section titles may be better as "Build", and "Run", respectively.
- Examples section: @emxsys mentioned keeping this section in an earlier comment, but I'm out of the loop on what that looked like initially. Is there any action needed to address this section?

**Comment by pdavidc** [comment]
Looks great!

**Review by pdavidc — APPROVED** [review]

**Pull request merged**